### PR TITLE
Add Fake News Detection in Filipino dataset

### PR DESCRIPTION
**Pull request by anaerobeth**
This PR adds the Fake News Filipino Dataset, a low-resource fake news detection corpora in Filipino. Contains 3,206 expertly-labeled news samples, half of which are real and half of which are fake.

Link to the paper: http://www.lrec-conf.org/proceedings/lrec2020/index.html

Link to the dataset/repo: https://github.com/jcblaisecruz02/Tagalog-fake-news

### CODE REVIEW

**Comment by jcblaisecruz02** [comment]
Hi! I'm the author of this paper (surprised to see our datasets have been added already).

That paper link only leads to the conference index, here's a link to the actual paper: https://www.aclweb.org/anthology/2020.lrec-1.316/

Would it be fine if I also edited your gsheet entry to reflect this change?

**Comment by anaerobeth** [comment]
Hi Jan, please go ahead and update. I see you are also in the sprint slack channel. Let me know if what else needs updating. Thanks.

File: datasets/fake_news_filipino/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+## Dataset Structure
+
+### Data Instances
+
+Sample data:
+{
+  "label": "0",
+  "article": "Sa 8-pahinang desisyon, pinaboran ng Sandiganbayan First Division ang petition for Writ of Preliminary Attachment/Garnishment na inihain ng prosekusyon laban sa mambabatas."
+}
```
Comment (lhoestq) [inline datasets/fake_news_filipino/README.md:75]:
````suggestion
```
{
  "label": "0",
  "article": "Sa 8-pahinang desisyon, pinaboran ng Sandiganbayan First Division ang petition for Writ of Preliminary Attachment/Garnishment na inihain ng prosekusyon laban sa mambabatas."
}
```
````

**Review by lhoestq — APPROVED** [review]
Awesome thanks !

File: datasets/fake_news_filipino/fake_news_filipino.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "label": datasets.features.ClassLabel(names=["0", "1"]),
+                "article": datasets.Value("string")
+            }
+        )
```
Comment (lhoestq) [inline datasets/fake_news_filipino/fake_news_filipino.py:56]:
```suggestion
        features = datasets.Features(
            {"label": datasets.features.ClassLabel(names=["0", "1"]), "article": datasets.Value("string")}
        )
```

File: datasets/fake_news_filipino/fake_news_filipino.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        data_dir = dl_manager.download_and_extract(_URL)
+        train_path = os.path.join(data_dir, "fakenews", "full.csv")
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": train_path,
+                    "split": "train",
+                }
```
Comment (lhoestq) [inline datasets/fake_news_filipino/fake_news_filipino.py:74]:
```suggestion
                },
```

**Pull request merged**